File: server/php/src/hz/message/message_creator.php
<?php

require_once('mysql_holder.php');
require_once('mysql_post.php');
require_once('message.php');

/**
 * 创建mysql类message
 *
 * @param string $user mysql用户名
 * @param string $password mysql用户密码
 * @param string $database mysql数据库名
 * @return Message message对象
 */
function createMessageByMysql($user, $password, $database) {
    $holder = new MysqlHolder($user, $password, $database);
    $post = new MysqlPost($holder->inst());
    return new Message($holder, $post);
}

?>

File: server/java/src/hz/message/MessageCreator.java
package hz.message;

import java.sql.Connection;

/**
 * message对象创建类
 */
public class MessageCreator {
	/**
	 * 创建mysql类message
	 * 
	 * @param user mysql用户名
	 * @param password mysql用户密码
	 * @param database mysql数据库名
	 * @return message对象
	 * @throws Exception 错误异常
	 */
	public static Message createMessageByMysql(String user, String password, String database) throws Exception {
		Holder holder = new MysqlHolder(user, password, database);
		Post post = new MysqlPost((Connection) holder.inst());
		return new Message(holder, post);
	}
}
